justfile

```diff
@@ -192,8 +192,8 @@ conformance-ignored:
     tmpfile="$(mktemp)"
     bash -c '[[ -n "$(git status -s)" ]] && echo "WARNING: uncommited changes will NOT be tested" || true'
     DNS_TEST_VERBOSE_DOCKER_BUILD=1 DNS_TEST_PEER=unbound DNS_TEST_SUBJECT="hickory {{justfile_directory()}}" cargo test --manifest-path conformance/Cargo.toml -p conformance-tests -- --ignored | tee "$tmpfile"
-    grep 'test result: FAILED. 0 passed' "$tmpfile" || ( echo "expected ALL tests to fail but at least one passed; the passing tests must be un-#[ignore]-d" && exit 1 )
     bash -c '[[ -n "$(git status -s)" ]] && echo "WARNING: uncommited changes were NOT tested" || true'
```

Comment:
I think there must be something else causing this to pass incorrectly, the [just documentation](https://just.systems/man/en/ignoring-errors.html) says that failures of commands other than the last one will stop recipes.

Reply:
Since this is a shebang recipe, just isn't involved in between the different commands that are being run. We can fix this by adding `set -e` at the start of the inline script.

Reply:
We should do the same with `e2e-tests-ignored` and `ede-dot-com-ignored` as well.